specification/servicefabricmanagedclusters/ServiceFabricManagedClusters.Management/NodeType.tsp

```diff
@@ -439,12 +439,17 @@ model NodeTypeProperties {
   vmSetupActions?: VmSetupAction[];
 
   /**
-   * Specifies the security type of the nodeType. Only Standard and TrustedLaunch are currently supported
+   * Specifies the security type of the nodeType. Supported values include Standard, TrustedLaunch and ConfidentialVM.
    */
   securityType?: SecurityType;
 
   /**
-   * Specifies whether secure boot should be enabled on the nodeType. Can only be used with TrustedLaunch SecurityType
+   * Specifies the security type of the nodeType. Only Standard and TrustedLaunch are currently supported.
```

Comment:
wrong description, should be like the VMSS swagger https://learn.microsoft.com/en-us/azure/templates/microsoft.compute/virtualmachinescalesets?pivots=deployment-language-arm-template#vmdisksecurityprofile-1